lisa/features/nvme.py

```diff
@@ -43,32 +30,13 @@ def enabled(self) -> bool:
         return True
 
     def get_devices(self) -> List[str]:
-        devices_list = []
-        self._get_device_from_ls()
-        for row in self._ls_devices.splitlines():
-            matched_result = self._device_pattern.match(row)
-            if matched_result:
-                devices_list.append(matched_result.group("device_name"))
-        return devices_list
+        return self._node.tools[Ls].get_nvme_devices()
```

Comment:
`Ls` should focus it's own logic on "list files". The nvme logic shouldn't be put in ls.